### ISSUE TITLE
Suggestion: Copy Scenario and if not conditions

### ISSUE THREAD

**Issue by Nfivez** [Nain57/Smart-AutoClicker#38]
Greetings! Love the app as one of the app I used before this broke. 

Copying scenario or clicks from other scenario will be a great addition to the app for me in one of the game I play as the condition are usually the same.

If not condition will also be great as I have to wait for people to join battle in the game to start.

I'll take a look if I can fork this and develop this myself but I don't have much experience developing this myself.

Thanks!

**Comment by Nain57** [comment]
Hi !

* "Copying scenario": This can be added. I will consider it for a future version.
* "If not condition will also be great": I don't get the wanted behaviour here. If a condition is not met, the clic is just not executed. Can you provide more information ?

**Comment by Nfivez** [comment]
Hmm, this might be a niche use case, but here is how it works for me. There is a loading icon when no one joins the lobby, the icon gets replaced by one of hundreds profile icon, then that is when I really start the lobby.

tldr is that I click when the icon changes but I can't really do different player icons. 

If this is difficult to implement, I understand. Cheers and best of luck!

**Comment by Nain57** [comment]
Now I understand the usage, thx.

The refactoring of the database in version 1.2 should allow me to implement those kind of options. I will consider it, but it will not come soon

**Comment by Nain57** [comment]
Copy feature implemented on last version: https://github.com/Nain57/Smart-AutoClicker/releases/tag/1.2.0
It should be available on F-Droid soon.

Condition negation might come in the future.

**Comment by Nfivez** [comment]
Alright, thanks! Copy scenario was more important for me, and condition negation while nice, I can wait. I shall open it maybe a later date.
